python.exe: can't open file '\\\\wsl.localhost\\Ubuntu\\mnt\\c\\Users\\Shaun Padrejuan\\Documents\\GitHub\\LabExercise3\\scripts\\branch_merged_data.py': [Errno 13] Permission denied
